src/main/java/racingcar/view/InputView.java

```diff
@@ -1,23 +1,26 @@
 package racingcar.view;
 
+import java.util.Arrays;
 import java.util.Scanner;
 
 public class InputView {
+    private static final Scanner scanner = new Scanner(System.in);
+
+    private static final String SEPARATOR = ",";
+    private static final String CAR_NAME_MESSAGE = "경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).";
+    private static final String GAME_ROUND_MESSAGE = "시도할 회수는 몇 회 인가요?";
 
-    private InputView(){
+    private InputView() {
         throw new AssertionError();
     }
-    private static final Scanner scanner = new Scanner(System.in);
 
-    private static String CAR_COUNT_MESSAGE = "자동차 대수는 몇 대 인가요?";
-    private static String GAME_ROUND_MESSAGE = "시도할 회수는 몇 회 인가요?";
-
-    public static int getCarCount() {
-        System.out.println(CAR_COUNT_MESSAGE);
-        return scanner.nextInt();
+    public static String[] getCarNames() {
```

Comment:
정확히는 차 이름들을 **입력**받는 메소드니까 아래처럼 바꾸는건 어떨까요?
```suggestion
    public static String[] inputCarNames() {
```
개인적으로 메소드 이름에 get- 을 붙이는건 일반 getter와 헷갈릴 여지가 있을 것 같습니다:)